dts/bindings/sensor/bosch,bmp180.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2024 Chris Ruehl
+# SPDX-License-Identifier: Apache-2.0
+
+# Common fields for BMP180
+
+include: sensor-device.yaml
+
+properties:
+  int-gpios:
+    type: phandle-array
+
+  osr-press:
+    type: int
+    description: |
+      Default pressure oversampling rate. Only the following values are
+      allowed:
+        0 ultra low power, 1 sample
+        1 standard,        2 sample
+        2 high,            4 sample
+        3 ultra high,      8 sample
+    default: 1
```

Comment:
The Bosch data-sheet mentioned the OSR "standard" so it seems to me that this is a candidate for default.
I will remove the default because it is not explicit noted anywhere.